### PR TITLE
Chess.com bot new version

### PR DESCRIPTION
**Pull request by Gothesque**
+ Button to start a new game
+ Autoplay
+ Config file (for lichess too)
+ More print so you know what is happening
* README changes

Still to do: Revert the board when you're black. You can also just flip the board in chess.com, but less effort is better 😄 

So yeah I added required things to make it life saving

### CODE REVIEW

**Comment by erxson** [comment]
Good job!

**Comment by erxson** [comment]
Thank you

**Comment by vaezim** [comment]
Thanks for the effort. I finally woke up from a two-month long sleep. Will write some review comments and then merge.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# 🤖 Online Chess Cheater Bot
-[lichess.org](https://lichess.org/) and [chess.com](https://chess.com) are the two most popular online chess platforms. This application provides the user with the top engine move and position evaluation. Once a live online game is started, it automatically pulls the played moves from the webpage's source and builds it own chess board. A local stockfish engine then analyzes the position and offers the best move to the user.
+# 🤖 Online Chess Bot
```
Comment (vaezim) [inline README.md:1]: let's not hide the real (evil) purpose of this project :()

Lichess has its own "Bot" players but here an actual human logs in, so...

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+[lichess.org](https://lichess.org/) and [chess.com](https://chess.com) are the two most popular online chess platforms. This application provides the user with the top engine move and position evaluation. Once a live online game is started, it automatically pulls the played moves from the webpage's source and helps you do a move. A local stockfish engine then analyzes the position and offers the best move to the user.
```
Comment (vaezim) [inline README.md:2]: how about:
builds a new chess board which indicates the best move to the user.

**Review by vaezim — COMMENTED** [review]
Adding the json config file was a good idea. will merge that for certain.
I will cherry-pick the remaining changes.

File: README.md (hunk 2 above)
Comment (Gothesque) [reply README.md:2]: So smth like 
```markdown
# 🤖 Online Chess Cheat Bot
[lichess.org](https://lichess.org/) and [chess.com](https://chess.com) are the two most popular online chess platforms. This application provides the user with the top engine move and position evaluation. Once a live online game is started, it automatically pulls the played moves from the webpage's source and shows on a new local chess board what is the best move to the user.```

**Pull request closed without merging**